app/models/exercise.rb

```diff
@@ -113,7 +115,7 @@ def to_resource_h
     language_resource_h = language.to_embedded_resource_h if language != guide.language
     as_json(only: %i(name layout editor corollary teacher_info manual_evaluation locale
                      choices assistance_rules randomizations tag_list extra_visible goal
-                     free_form_editor_source initial_state final_state))
+                     free_form_editor_source initial_state final_state settings))
```

Comment:
Warning, you may be including the guide's settings here. Merge them seperately like with the expectations